components/brave_vpn/resources/panel/vpn_panel.tsx

```diff
@@ -50,7 +39,10 @@ function App () {
 
 function initialize () {
   render(<App />, document.getElementById('mountPoint'),
-  () => apiProxy.getInstance().showUI())
+  () => {
+    apiProxy.getInstance().showUI()
+    apiProxy.getInstance().createVPNConnection()
```

Comment:
@petemill Added this for now and will replace it in the future. Eventually, we're figuring out if webui should control creating/deleting vpn connection or not. 

In-order to switch regions we need to remove the current profile and add a new profile and @simonhong are discussing on how to go about controlling this.

Reply:
On macOS, I found that we don't need to delete existing entry.
Instead, updating it with new region info and current connection will be disconnected if was connected.
Then, we have to ask connect again and next connection will use new vpn configuration.